src/rubyfmt.rb

```diff
@@ -716,6 +716,25 @@ def format_int(ps, rest)
   ps.emit_indent if ps.start_of_line.last
 
   int = rest[0]
+
+  if int[0] != "0"
+    int_array = []
+    counter = 0
+
+    int.chars.reverse.each do |char|
+      next if char == "_"
```

Comment:
I'd use `#reject` to remove the chars that include `_` before looping around with `#each`.

But I'm unsure how @samphippen feels about that.

Reply:
Definitely could do that, I was just trying to optimize doing this in a single iteration over the string.